src/promptflow-devkit/promptflow/_sdk/_service/static/trace/index.html

```diff
@@ -1,16 +1,16 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="icon" href="/v1.0/ui/traces/assets/icon-oxB8RKmI.svg" media="(prefers-color-scheme:no-preference)">
-    <link rel="icon" href="/v1.0/ui/traces/assets/icon_for_dark-7fdboOJT.svg"  media="(prefers-color-scheme:dark)">
-    <link rel="icon" href="/v1.0/ui/traces/assets/icon-oxB8RKmI.svg" media="(prefers-color-scheme:light)">
-    <title>Trace View</title>
-    <script type="module" crossorigin src="/v1.0/ui/traces/assets/index-uhZP70pN.js"></script>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="/v1.0/ui/traces/assets/icon-IVYk8x5p.svg" media="(prefers-color-scheme:no-preference)">
+    <link rel="icon" href="/v1.0/ui/traces/assets/icon_for_dark-3C8HbOu4.svg"  media="(prefers-color-scheme:dark)">
+    <link rel="icon" href="/v1.0/ui/traces/assets/icon-IVYk8x5p.svg" media="(prefers-color-scheme:light)">
+    <title>Trace View</title>
+    <script type="module" crossorigin src="/v1.0/ui/traces/assets/index-Gv8oRpaG.js"></script>
```

Comment:
update CHANGELOG for this change

Reply:
FYI.

- devkit: https://github.com/microsoft/promptflow/blob/main/src/promptflow-devkit/CHANGELOG.md
- pf: https://github.com/microsoft/promptflow/blob/main/src/promptflow/CHANGELOG.md